build-system/eslint-rules/no-import.js

```diff
@@ -41,14 +41,13 @@ module.exports = function(context) {
       const name = node.source.value;
 
       for (const forbidden of imports) {
-        if (name !== forbidden.import) {
-          continue;
+        const importSource = forbidden.import;
+        if (name === importSource || name.startsWith(`${importSource}/`)) {
```

Comment:
Can you now delete this rule?

https://github.com/ampproject/amphtml/blob/0b9623cbc43521e28e395a58b9ab3613044cda14/build-system/eslint-rules/no-import.js#L25-L29

Reply:
@jridgewell Was this rule still needed?  With this change, isn't the `preact` rule sufficient?